test/plots/rectanglePlotTests.ts

```diff
@@ -77,6 +77,44 @@ describe("Plots", () => {
       svg.remove();
     });
 
+    it("retrieves the correct entities intersect with the bounding box", () => {
+      let xScale = new Plottable.Scales.Linear();
+      let yScale = new Plottable.Scales.Linear();
+      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      let dataset = new Plottable.Dataset(DATA);
+      let plot = new Plottable.Plots.Rectangle()
+        .x((d) => d.x, xScale).x2((d) => d.x2)
+        .y((d) => d.y, yScale).y2((d) => d.y2);
+      plot.addDataset(dataset).renderTo(svg);
+
+      let entities = plot.entitiesIn({
+        topLeft: { x: xScale.scale(1.5), y: yScale.scale(2.5) },
+        bottomRight: { x: xScale.scale(2.5), y: yScale.scale(1.5) } });
+      assert.lengthOf(entities, 2, "retrieved 2 entities intersect with the box");
+      assert.strictEqual(entities[0].index, 1, "second entity is retrieved");
```

Comment:
Maybe we should say "the 1-th entity is retrieved", even though that's not grammatically correct, since with the current setup we'll get the kind of odd message:
```
second entity is retrieved: expected <number> to equal 1
```